<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeBox</class>
 <widget class="QWidget" name="CodeBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>342</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#CodeBox{
	background-color: transparent;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="TabBarPlaceholder" native="true"/>
   </item>
   <item>
    <widget class="QFrame" name="Background">
     <property name="styleSheet">
      <string notr="true">#Background{
	background-color: rgb(255, 245, 239);
	border-top-left-radius: 26px;
	border-top-right-radius: 26px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_8">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_7">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>26</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>26</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#frame_7{
	background-color: rgb(255, 155, 82);
	border-top-left-radius: 26px;
	border-top-right-radius: 26px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="StackedWidget">
        <property name="styleSheet">
         <string notr="true">#CodeStackedWidget{
	background-color: transparent;
}
</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
